Doc/library/collections.rst

```diff
@@ -1161,6 +1161,8 @@ variants of :func:`functools.lru_cache`::
             return value
 
         def __setitem__(self, key, value):
+            if key in self:
+                self.move_to_end(key)
             super().__setitem__(key, value)
             if len(self) > self.maxsize:
```

Comment:
I think this part could be replaced by `self.popitem(last=False)` now.

Reply:
Yes, you are right.
It is cleaner to remove the oldest entry by `self.popitem(last=False)`.

At the same time, It is working correctly.

Do you suggest I update it in this pull request?
Not sure about the conventions here.
Normally we are only supposed to change the code directly related to the issue.

Reply:
I don't know, I think `popitem()` was added in Python3 so this block of code needed to be compatible with Python 2 until this year.

Let's wait for @rhettinger review, I can change it in another PR if he thinks it's needed.